src/components/sdk/MeetingControls/VideoInputControl.tsx

```diff
@@ -15,13 +15,17 @@ import useSelectVideoInputDevice from '../../../hooks/sdk/useSelectVideoInputDev
 interface Props {
   /** The label that will be shown for video input control, it defaults to `Video`. */
   label?: string;
+  /** A boolean that determines whether or not to include additional sample video devices to the control bar */
+  appendAdditionalDevices?: boolean;
 }
 
-const videoInputConfig: DeviceConfig = {
-  additionalDevices: true,
-};
-
-const VideoInputControl: React.FC<Props> = ({ label = 'Video' }) => {
+const VideoInputControl: React.FC<Props> = ({ 
+  label = 'Video',
+  appendAdditionalDevices = true,
+}) => {
+  const videoInputConfig: DeviceConfig = {
+    additionalDevices: appendAdditionalDevices,
```

Comment:
What will be the impact if this is made configurable ?

Reply:
Ideally, the user can decide whether or not they want these additional devices in the input controls.